ROADMAP.md

```diff
@@ -10,13 +10,16 @@ roadmap](https://github.com/libp2p/specs/blob/master/ROADMAP.md).
 
 ## TODO and in-progress
 
-### Attempt to switch from webrtc-rs to str0m
+### WebTransport
 
-| Category     | Status | Target Completion | Tracking                                          | Dependencies | Dependents |
-|--------------|--------|-------------------|---------------------------------------------------|--------------|------------|
-| Connectivity | todo   |                   | https://github.com/libp2p/rust-libp2p/issues/3659 |              |            |
+| Category                    | Status | Target Completion | Tracking                                          | Dependencies                       | Dependents |
+|-----------------------------|--------|-------------------|---------------------------------------------------|------------------------------------|------------|
+| Connectivity / optimization | todo   |                   | https://github.com/libp2p/rust-libp2p/issues/2993 | [QUIC](#experimental-quic-support) |            |
 
-Reduce maintenance burden and reduce dependency footprint.
+A WebTransport implementation in rust-libp2p will enable browsers to connect to rust-libp2p nodes
+where the latter only have a self-signed TLS certificate. Compared to WebRTC, this would likely be
+more performant. It is dependent on QUIC support in rust-libp2p. Given that we will support WebRTC
+(browser-to-server) this is not a high priority.
```

Comment:
Mind switching to one sentence per line?

Plus, I given that we have QUIC now, I think we line about the QUIC dependency can be removed.

Reply:
Also, the line about the priority seems to be outdated now :)